IDEAS/Buildings/Components/Interfaces/PartialSurface.mo

```diff
@@ -260,15 +354,20 @@ model Q50_parameterToConnector "Converts parameter values into connectors for pr
   Modelica.Blocks.Interfaces.BooleanOutput using_custom_q50 = use_custom_q50
     "Output indicating whether a custom q50 value should be considered by the zone"
    annotation (Placement(transformation(extent={{-100,-30},{-120,-10}})));
+  Modelica.Blocks.Interfaces.RealInput dummy_h[2]
```

Comment:
Can you add a comment why we need this RealInput? (to avoid a warning in Dymola)